src/components/UI/Overlay/Overlay.tsx

```diff
@@ -130,6 +298,16 @@ export const OverlayAddressBook = ({ addressBook = {}, ...props }: OverlayProps)
           </tbody>
         </table>
       </div>
-    </Overlay>
+    </OverlayDefault>
+  );
+};
+
+export const OverlayYoutube = ({ youtubeId, ...props }: OverlayProps) => {
```

Comment:
I would suggest to remove youtubeId from OverlayProps and use the & operator from typescript or to extends OverlayProps:

1. 
```
export const OverlayYoutube = ({ youtubeId, ...props }: OverlayProps & {youtubeId: string})
```

2.
```
interface OverlayYoutubeProps extends OverlayProps{
  youtubeId: string;
}
```

Reply:
nice ok